app/xeve_app.c

```diff
@@ -633,6 +633,261 @@ static int update_rc_param(ARGS_PARSER * args, XEVE_PARAM * param)
     return 0;
 }
 
+
+static int vui_param_check(XEVE_PARAM * param)
```

Comment:
Where do you call this function?

Reply:
Added the call, I think it was deleted out while revert and again pushing it back. pls check now